react/MonthPicker/CustomMonthPicker/CustomMonthPicker.js

```diff
@@ -34,16 +35,14 @@ export default class CustomMonthPicker extends Component {
       year: PropTypes.number
     }),
     valid: PropTypes.bool,
-    className: PropTypes.string,
     id: PropTypes.string,
     minYear: PropTypes.number.isRequired,
     maxYear: PropTypes.number.isRequired,
     ascendingYears: PropTypes.bool.isRequired
   };
 
   static defaultProps = {
-    value: {},
-    className: ''
```

Comment:
This is technically a breaking change. Why is this being removed?

Reply:
Sorry, I missed that this change isn't actually part of the public API. All good 👍